52.7.0 CLI is broken due to missing dist/cli.js

<!-- ⚠️ Please DON'T ignore the issue template -->

<!-- 💡 Consider upgrading to the latest version before sending the issue -->

**Describe the bug**

After updating Slidev to 52.7.0, the `slidev` cli command fails with the following error:

```
$ pnpm slidev
node:internal/modules/esm/resolve:274
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/ryotaushio/code/slidev-repro/node_modules/.pnpm/@slidev+cli@52.7.0_@babel+parser@7.28.5_@nuxt+kit@3.20.1_@types+markdown-it@14.1.2_@typ_e101fd0a655c091c4cf30e4c899c3e89/node_modules/@slidev/cli/dist/cli.js' imported from /Users/ryotaushio/code/slidev-repro/node_modules/.pnpm/@slidev+cli@52.7.0_@babel+parser@7.28.5_@nuxt+kit@3.20.1_@types+markdown-it@14.1.2_@typ_e101fd0a655c091c4cf30e4c899c3e89/node_modules/@slidev/cli/bin/slidev.mjs
    at finalizeResolution (node:internal/modules/esm/resolve:274:11)
    at moduleResolve (node:internal/modules/esm/resolve:859:10)
    at defaultResolve (node:internal/modules/esm/resolve:983:11)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:731:20)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:708:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:310:38)
    at onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:664:36)
    at TracingChannel.tracePromise (node:diagnostics_channel:350:14)
    at ModuleLoader.import (node:internal/modules/esm/loader:663:21)
    at defaultImportModuleDynamicallyForModule (node:internal/modules/esm/utils:222:31) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///Users/ryotaushio/code/slidev-repro/node_modules/.pnpm/@slidev+cli@52.7.0_@babel+parser@7.28.5_@nuxt+kit@3.20.1_@types+markdown-it@14.1.2_@typ_e101fd0a655c091c4cf30e4c899c3e89/node_modules/@slidev/cli/dist/cli.js'
}

Node.js v22.21.0
```

Apparently, the slidev 52.7.0 has transitioned to esm-only and lacks `dist/cli.js`( which had been there until the previous release).
However, the import here still expects `cli.js` instead of `cli.mjs`:

https://github.com/slidevjs/slidev/blob/822031cfc266ec297904f6dac343def18abc7b70/packages/slidev/bin/slidev.mjs#L3

(Just updating this import might fix the issue but I'll have to take a deeper look to verify it.)

**Minimal reproduction**

Steps to reproduce the behavior:

Just initialize a project with `pnpm create slidev@52.7.0`.

After installing dependencies, the "create" command tries to run `slidev --open`, but it fails and you will see the error.

**Environment**

- Slidev version: 52.7.0
- Browser: Not relevant
- OS: macOS 15.5
- Node.js: 22.21.0

--- a/packages/slidev/bin/slidev.mjs
+++ b/packages/slidev/bin/slidev.mjs
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 'use strict'
-import ('../dist/cli.js')
+import('../dist/cli.mjs')

